src/mctpd.c

```diff
@@ -2425,6 +2654,26 @@ static int bus_endpoint_find_uuid(sd_bus *bus, const char *path,
 	return 0;
 }
 
+/* Association.Definitions interface is only added for peers that get EID
+   configuration from entity-manager */
+static int bus_endpoint_find_association(sd_bus *bus, const char *path,
+	const char *interface, void *userdata, void **ret_found,
+	sd_bus_error *ret_error)
+{
+	ctx *ctx = userdata;
+	peer *peer = NULL;
+	int rc;
+
+	rc = peer_from_path(ctx, path, &peer);
+	if (rc >= 0 && peer->published) {
+		if (peer->association) {
```

Comment:
just include this in the conditional above maybe?

Reply:
Done